stories/core/Tree.stories.tsx

```diff
@@ -0,0 +1,139 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import { Tree, TreeProps, TreeNode, Checkbox } from '../../src/core';
+import { action } from '@storybook/addon-actions';
+import { SvgPlaceholder } from '@itwin/itwinui-icons-react';
+import { NodeData } from '../../src/core/Tree/Tree';
+
+export default {
```

Comment:
Add 
![image](https://user-images.githubusercontent.com/36186912/150942178-4b3343a8-392d-48e3-89a6-969e6361e9d7.png)
This hides warning message in story.

Reply:
Added